src/components/editor/EditorArticleController.jsx

```diff
@@ -35,6 +35,7 @@ export default class EditorArticleController extends FalcorController {
         // As changedFlag is currently false, we can simply use equals here
         changedFlag = fields.some((field) => {
           let formValue = formNode[field].value;
+          console.log(typeof formValue);
           if (field === "issue") {
             formValue = parseInt(formValue);
```

Comment:
Yeah it's true. I checked that though, and right now I am actually giving it an integer value. But HTML automatically converted it to a string before it is passed in as the value.